.gitignore

```diff
@@ -20,4 +20,6 @@ gha-creds-*.json
 .idea
 scrap
 output/*
-imgs/*
\ No newline at end of file
+imgs/*
+# don't track catboost training info
+*/catboost_info
```

Comment:
`catboost_info` looks like it is in git - should that folder be removed from git?

Reply:
It should indeed not be in git.